angular-client/package.json

```diff
@@ -31,23 +33,33 @@
     "apexcharts": "^3.44.0",
     "mapbox-gl": "^3.1.0",
     "material-design-icons": "^3.0.1",
-    "ng-apexcharts": "^1.8.0",
+    "ng-apexcharts": "1.8.0",
```

Comment:
why no more updates to compatible versions?

Reply:
This was giving me some annoying dependency errors, removing it was the quickest fix